src/content/structured/components/dialog/guidance.mdx

```diff
@@ -177,3 +177,5 @@ Always follow the [content style](/styles/content-style) when writing textual co
   state="none"
   caption="Dialogs have three sections: a title area, a content area and an action area."
 />
+
+When adding interactive content to a dialog and it has popover elements, to avoid the elements being cut off within the content area, use the prop `disableHeightConstraint` to allow the dialog to stretch to contain it's content.
```

Comment:
incorrect use of apostrophe in `contain it's content`, should be `contain its content`

sorry for being so pedantic! [justification from merriam webster](https://www.merriam-webster.com/grammar/when-to-use-its-vs-its)